src/SvelteTable.svelte

```diff
@@ -75,6 +92,36 @@
   const handleClickCell = (event, row, key) => {
     dispatch('clickCell', { event, row, key });
   };
+
+  const handleClickPage = (direction) => {
+    switch (direction) {
+      case 'First':
```

Comment:
Nitpick: I prefer using a constant or enum variable (whatever you prefer more) instead of text. It reduces the risk of misspelling. You may count it as my personal whim. :)